src/main/java/book/store/model/Book.java

```diff
@@ -0,0 +1,31 @@
+package book.store.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Entity
+@Data
```

Comment:
don't use `@Data` for entities
```suggestion
@Getter
@Setter
```